server.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable prefer-import/prefer-import-over-require */
+
+const path = require('path');
+const express = require('express');
+const next = require('next');
+const compression = require('compression');
+const expressStaticGzip = require('express-static-gzip');
+
+const host = process.env.HOST || '127.0.0.1';
```

Comment:
Having this will keep us from accessing our IP in the same network which is very useful when testing in other devices.

Reply:
Maybe we can change the default to `0.0.0.0`?